examples/node/lib/rest/versionless/deployed_devices/fleet.ts

```diff
@@ -236,6 +237,8 @@ export interface FleetListInstance {
   [inspect.custom](_depth: any, options: InspectOptions): any;
 }
 
+export interface FleetSolution {}
```

Comment:
How come we're still exporting solution types even if there's no path params for it?

Reply:
It's just generic in that we export the solution type used by the instance, even though some have no properties in the solution.